### DISCUSSION TITLE
What are the reasons Pgbouncer Exporter is updated much less frequently than other images like Pgbouncer itself?

### DISCUSSION THREAD

**Discussion by andrii-korotkov-verkada** [apache/airflow#45005] [Q&A] [answered]
Hello. I hope your day is going well. It seems that Pgbouncer Exporter is updated 1-2 times per year, as opposed to other images which seem to be updated every 1-2 months or so. Our security team would prefer a usage of fresher images if possible. What's the reason it's not updated as frequently? Can I contribute somehow to improve this? Thank you.

**Comment by potiuk** [comment, marked as answer]
Sure - there is no particular reason - mostly, it's updated when someone updates it - because this is open-source and this is  how it works. Yes - you can absolutely contribute to it - just make a PR similar to https://github.com/apache/airflow/pull/40318 - then one of the maintainers (likely me) will refresh the images and merge the PR).
